### DISCUSSION TITLE
Error: You cannot use different slug names for the same dynamic path

### DISCUSSION THREAD

**Discussion by abhishekdarteweb** [vercel/next.js#17402] [Help]
Hello,

Please help me to resolve this issue TIA

![image](https://user-images.githubusercontent.com/26562926/94392649-7ecc0280-0176-11eb-9f15-8e6232f38703.png)

I'm getting this error when run command  npm run dev 
Error: You cannot use different slug names for the same dynamic path ('single' !== 'folder').
for folder structure i have  attached a screenshot

**Comment by lfades** [comment]
@abhishekdarteweb Can you share a small reproduction of the issue or provide more details about it? Your path structure looks okay to me but that error is due to conflicts with the file names of your pages.

**Reply by kmcaloon** [reply]
@abhishekdarteweb you probably spotted the problem but just in case anyone else is having the same problem, I'm pretty sure the issue is that the `[folder]` directory needs to be renamed to `[single]`. For some reason Next needs the actual names to match.
